### ISSUE TITLE
Stack Termination protection

### ISSUE THREAD

**Issue by joehellmersNOAA** [aws/aws-parallelcluster#4155]
Is there a configuration option to enable Cfn stack termination protection in the PCluster configuration file?

**Comment by demartinofra** [comment]
Hi @joehellmersNOAA , no there is no way to do it from pcluster config file at the moment. You'd have to set it after cluster creation started by running something like `aws cloudformation update-termination-protection --stack-name <CLUSTER_NAME> --enable-termination-protection`

**Comment by joehellmersNOAA** [comment]
Thanks @demartinofra. I actually embedded my cluster create command in a shell script that does exactly that.

Can I put a request to make this enhancement?  It could either be an option in the configuration file, or it could be an option on the pcluster create-cluster command.

**Comment by demartinofra** [comment]
Yes, I think it makes total sense to have this as a create-cluster command option 👍🏻 Marking this as a feature request
